src/services/codefixes/fixAddMissingNewOperator.ts

```diff
@@ -0,0 +1,29 @@
+/* @internal */
+namespace ts.codefix {
+    const fixId = "addMissingNewOperator";
+    const errorCodes = [Diagnostics.Value_of_type_0_is_not_callable_Did_you_mean_to_include_new.code];
+    registerCodeFix({
+        errorCodes,
+        getCodeActions(context) {
+            const { sourceFile, span } = context;
+            const identifierWithoutNew = getIdentifier(sourceFile, span.start);

```

Comment:
This is not guaranteed to be an identifier - you really have an arbitrary expression. For example:

```ts
class C {}
let x = (() => C)()();
```

Can you add a respective test that ensures it gets corrected to

```ts
let x = new ((() => C)())();
```

Reply:
This indeed doesn't work at the moment.
I tried working with an expression here but it seems that the token isn't one:
`Verbose Debug Information: Node OpenParenToken did not pass test 'isExpression'`

When I tried working with its parent element, I received the following:
`new (() => C)()()` (Missing parentheses after the `new` addition). 
Some help would be very appreciated.